index.html

```diff
@@ -1,74 +1,400 @@
+            </ul>
+        </div>
+
+        <div class="card">
+            <img class="logo-img" src="./images/Group.svg" alt="">
+            <h2>
+                Frameworks
+            </h2>
+            <ul class="eylemm">
+                <li><button class="btn2">Bootstrap</button></li>
+                <li><button class="btn2">Ruby on rails</button></li>
+                <li><button class="btn2">RSpec</button></li>
+                <li><button class="btn2">Capybara</button></li>
+                <li><button class="btn2">Selenium</button></li>
+            </ul>
+        </div>
+
+        <div class="card">
+            <img class="logo-img1" src="./images/icon-skills.svg" alt="">
+            <h2>
+                Skills
+            </h2>
+            <ul class="eylemm">
+                <li><button class="btn2">Codekit</button></li>
+                <li><button class="btn2">GitHub</button></li>
+                <li><button class="btn2">Codepen</button></li>
+                <li><button class="btn2">Gitlab</button></li>
+                <li><button class="btn2">Terminal</button></li>
+            </ul>
+        </div>
+    </section>
+
+    <section class="enginn">
+        <p class="last-p">
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </p>
+        <div>
+            <form>
+                <input class="hover" placeholder="Full name">
+                <input class="hover" placeholder="Email address">
+                <textarea class="hover" placeholder="Enter text here"></textarea>
+                <button class="btngreen">Get in touch</button>
+            </form>
+        </div>
+        <div class="img2"></div>
+    </section>
+    </div>
+
+    <div class="addline_social">
+        <ul class="social_links1">
+            <li class="social_link">
+                <a href="#">
+                    <img class="social_icon" src="images/twitter.svg" alt="twitter">
+                </a>
+            </li>
+            <li class="social_links">
+            <li class="social_link">
+                <a href="#">
+                    <img class="social_icon" src="images/Linkedin icon.svg" alt="linkedin">
+                </a>
+            </li>
+            <li class="social_link">
+                <a href="#">
+                    <img class="social_icon" src="images/github.svg" alt="github">
+                </a>
+            </li>
+            <li class="social_link">
+                <a href="#">
+                    <img class="social_icon" src="images/medium.svg" alt="M">
+                </a>
+            </li>
+            <li class="social_link">
+                <a href="#">
+                    <img class="social_icon" src="images/angellist.svg" alt="Hand">
+                </a>
+            </li>
+        </ul>
+    </div>
+    <a class="link" href="#">cansu8tas@gmail.com</a>
+</body>
+
+</html>
```

Comment:
- [ ] Kudos for making some changes required in the previous review but you still need to improve on something, don't worry you are almost there, I noticed that your design didn't really fit the Figma design if we compare it in the screenshot below, kindly fix this :wink:  your forms didn't really look the same as Figma design, if you take an observation you see that we do have line bar at the end of our page, kindly work on that as well
 
Your design 
![loop](https://user-images.githubusercontent.com/88894525/206862498-fc89f9a7-3c19-4f53-a2b4-848a3da3e1d4.png)


Figma design 
![form12](https://user-images.githubusercontent.com/88894525/206862422-56f1737d-4aa7-4b2f-adff-b4db18255a09.png)